packages/localize/README.md

```diff
@@ -224,15 +222,14 @@ lit-localize supports two output modes: _transform_ and _runtime_.
 
 The `@lit/localize` module exports the following functions:
 
-> Note that lit-localize relies on distinctive, annotated TypeScript type
-> signatures to identify calls to `msg` and other APIs during analysis of your
-> code. Casting a lit-localize function to a type that does not include its
-> annotation will prevent lit-localize from being able to extract and transform
-> templates from your application. For example, a cast like
-> `(msg as any)("Hello")` will not be identified. It is safe to re-assign
-> lit-localize functions or pass them as parameters, as long as the distinctive
-> type signature is preserved. If needed, you can reference each function's
-> distinctive type with e.g. `typeof msg`.
+> Note that lit-localize relies on distinctive, annotated type signatures to
+> identify calls to `msg` and other APIs during analysis of your code. Casting a
```

Comment:
Casting is a TS concept. What's the equivalent guidance for JS?

Reply:
You can use a `/** @type import("@lit/localize").msg */` JSDoc comment.

I've actually moved this whole section to an FAQ item at the bottom of this README, and added examples of how to do the type cast in JS and TS. I don't think this issue actually needs to be so prominent, since it's a bit distracting and scary, and it only applies if you're doing something outside of our documentation that should be pretty unusual -- and even then most simple re-export and re-assign patterns will work fine in JS and TS.